Library/Formula/llvm.rb

```diff
@@ -45,9 +60,9 @@ class Llvm < Formula
       sha256 "940dc96b64919b7dbf32c37e0e1d1fc88cc18e1d4b3acf1e7dfe5a46eb6523a9"
     end
 
-    resource "clang-tools-extra" do
-      url "http://llvm.org/releases/3.6.2/clang-tools-extra-3.6.2.src.tar.xz"
-      sha256 "6a0ec627d398f501ddf347060f7a2ccea4802b2494f1d4fd7bda3e0442d04feb"
+    resource "polly" do
```

Comment:
Just to complement [my comment](https://github.com/Homebrew/homebrew/pull/46721#issuecomment-164157454): Remove this resource.